src/messages/ccls_dataFlowInto.cc

```diff
@@ -0,0 +1,97 @@
+/* Copyright 2017-2018 ccls Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "message_handler.hh"
+#include "pipeline.hh"
+#include "query.hh"
+
+#include <unordered_set>
+
+namespace ccls {
+namespace {
+struct Param : TextDocumentPositionParam {
```

Comment:
There is no need to define a new struct.

That was the purpose of commit 123f504d50e451bf84f71e109f230f220928fa4f. Before it, each method required a different parameter type.

Reply:
Then we'd have to move its REFLECT_STRUCT into message_handler.hh, otherwise the reflection below won't work.